extension/content.css

```diff
@@ -6,30 +6,32 @@ readerease-sentence {
 	border-radius: 3px;
 }
 
-.persistentHighlight {
+.persistentHighlight.persistentHighlight {
```

Comment:
this double thing is probably an accident?
Also -- pls check if the other changes in this file are intended.

Reply:
So it's oddly not, it adds more specificity to the CSS which lets it's "!important" override the important of the background. When the user highlights a sentence, it lets the background change color giving an indication that the highlight worked (otherwise it is very confusing, as it appears the highlighting didn't work)